src/constants/allowedFilterSettings.ts

```diff
@@ -0,0 +1,8 @@
+/** The allowed filter settings */
+export const ALLOWED_FILTER_SETTINGS = [
```

Comment:
I think `FILTER_SETTINGS` makes more sense for more general use.

Reply:
done in 7114288b7fa206203c02b45725545404c76f389b